src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -59,6 +376,40 @@ def __init__(self, sc, *args):
             self.ml = sc._jvm.org.apache.sysml.api.MLContext(sc._jsc, monitorPerformance, setForcedSparkExecType)
```

Comment:
style: Why not change the class instantiation to

```python
def __init__(self, sc, monitor=False, forceSpark=False):
```
Basically makes sure that passing args of `length>2` is not silently ignored.